examples/react-example/src/ui/App.tsx

```diff
@@ -135,7 +144,8 @@ export const App: React.FC = () => {
           Transfer
         </button>
       </div>
-
+      {/* <WagmiCoreProvider> */}
```

Comment:
please remove commented code here

Reply:
@salavat-gainetdinov-mesh please review the latest changes not the changes from yesterday.  Thank you.